scripts/apps/archive/controllers/ArchiveListController.js

```diff
@@ -125,6 +125,7 @@ export class ArchiveListController extends BaseListController {
         $scope.$on('content:update', refreshItems);
         $scope.$on('item:deleted', refreshItems);
         $scope.$on('item:highlight', refreshItems);
+        $scope.$on('item:marked_desks', refreshItems);
```

Comment:
Just FYI, not sure if you knew this, or if it applies here, but you can (and imo should) document events using `@ngdoc event` directive like [here](https://github.com/wingedfox/angular-route-segment/blob/master/src/route-segment.js#L679-L687). They should probably be documented at their origin though (where they are broadcasted).